config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,16 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+#: --tmpfs parameter
+tmpfs_path = /run
+run_options_csv = --tmpfs %(tmpfs_path)s
+#: command to run in container to check if passed --tmpfs is mounted as tmpfs type
+fndmnt = findmnt -n -t tmpfs %(tmpfs_path)s
+cmd = 'sleep 2s && %(fndmnt)s'
+run_append_name = True
```

Comment:
When in doubt, choose the path which requires the least amount of work.  Pressing the delete-key several times is constituted as work in my book :grinning: 

Seriously though, clarity is important, and IMHO removing it really doesn't save a whole lot of typing.  Unless you remove it from all the other subtests here.  But that's even more work.  So I say just leave it.